README.md

```diff
@@ -1,62 +1,110 @@
-# ROS2 workspace for PX4
+# ROS2 & PX4 CustomMode example
+This tutorial is a example on how to utilize custom modes in PX4 using ROS2 and QGC. We would like to provide you with a basic example code that you can customize and utilize for your application.
 
-## Pre-requisites
+This project is a customized example of the usage of the Auterions PX4-ROS2 Interface Library
+
+https://github.com/Auterion/px4-ros2-interface-lib
+
+### Prerequisites
+* Ubuntu 22.04
+* ROS2 Humble
+* PX4 Autopilot
+* Micro XRCE-DDS Agent
+* QGroundControl Daily Build 
+
+You can find the required instructions collected below
+
+https://docs.px4.io/main/en/ros2/user_guide.html
+
+
+https://docs.qgroundcontrol.com/master/en/qgc-user-guide/releases/daily_builds.html
+
+
+## Usage
+
+### Setup the Workspace
+Navigate to the directory you would like to place the worskpace and then run the following
+```
+git clone https://github.com/ARK-Electronics/ros2_desktop_ws
```

Comment:
go ahead and change the name to **px4_ros2_examples_ws** and I'll update the repository name.